src/AstTypeAnalysis.cpp

```diff
@@ -125,7 +125,7 @@ class AstConstraintAnalysis : public AstVisitor<void> {
      * @param debug a flag enabling the printing of debug information
      * @return an assignment mapping a property to each argument in the given clause
      */
-    solution_type analyse(const AstClause& clause, bool debug = false) {
+    solution_type analyse(const AstClause& clause, std::ostream* debug = NULL) {
```

Comment:
Use nullptr instead of NULL

Reply:
Fixed in e16185d10dee7789f65b1c444dce58994ab769ea